game-server/data/handlers/ai/instance/empyreanCrucible/RM1337AI.java

```diff
@@ -59,99 +59,64 @@ public void handleBackHome() {
 	@Override
 	public void handleAttack(Creature creature) {
 		super.handleAttack(creature);
-		if (isHome.compareAndSet(true, false)) {
+		if (isHome.compareAndSet(true, false))
 			startSkillTask1();
-		}
-		if (getLifeStats().getHpPercentage() <= 75) {
-			if (isEventStarted.compareAndSet(false, true)) {
-				startSkillTask2();
+		handleHpPhase(getLifeStats().getHpPercentage());
```

Comment:
This method should not be called directly. Refer to https://github.com/beyond-aion/aion-server/pull/5#discussion_r1695500455 for the correct usage.